third_party/packages/split/lib/split.dart

```diff
@@ -88,31 +107,80 @@ Splitter flexSplit(
       gutterSize: gutterSize,
       sizes: sizes,
       minSize: minSize,
+      expandToMin: minSize?.isNotEmpty == true,
     ),
   );
 }
 
-/// Creates a splitter that changes from horizontal to vertical depending on the
-/// window aspect ratio.
+/// Splitter that splits multiple elements that must have a parent of fixed
+/// width.
```

Comment:
nit: replace width with size. The splitter could be splitting elements vertically or horizontally

Reply:
done